File: commands/Admin/src/main/java/be/perzival/dev/danager/commands/admin/PingCommand.java
package be.perzival.dev.danager.commands.admin;

import de.btobastian.sdcf4j.Command;
import de.btobastian.sdcf4j.CommandExecutor;
import org.springframework.stereotype.Component;

@Component
public class PingCommand implements CommandExecutor {

    @Command(aliases = {"ping"}, description = "pong!", usage = "ping")
    public String onCommand() {
        return "Pong!";
    }
}

File: commands/Admin/src/main/kotlin/be/perzival/dev/danager/commands/admin/PingCommand.kt
package be.perzival.dev.danager.commands.admin

import de.btobastian.sdcf4j.Command
import de.btobastian.sdcf4j.CommandExecutor
import org.springframework.stereotype.Component

@Component
class PingCommand : CommandExecutor {
    @Command(aliases = ["ping"], description = "pong!", usage = "ping")
    fun onCommand(): String = "Pong!"
}